maint/bumplibs.sh

```diff
@@ -190,11 +190,14 @@ process_lib() {
         prompt_to_continue
     fi
 
-    # Show merged PRs since last release touching this lib
-    echo ""
-    echo "- PRs merged touching lib$LIB since $LAST_RELEASE:"
-    git log Pacemaker-2.0.3..HEAD -z $HEADERS_HEAD $SOURCES $AMFILE|grep -z "Merge pull request"|sed -zr "s/.*#([0-9]+).*/#\1 /"
-    echo ""
+    # @TODO this seems broken ...
```

Comment:
Yep ... that slipped when I tried to make stuff I had hacked for 2.0.4-release generic.
Just tried on the cmdline and it did work as expected - create a single line with all PR-numbers that is compact enough that it doesn't need a query.
Maybe dependency to versions ... Tried on git version 1.8.3.1.

Reply:
For example, with either git 1.8.3.1 or 2.27.0, either of these:

    git log Pacemaker-2.0.4..HEAD -z configure.ac|grep -z "Merge pull request"|sed -zr "s/.*#([0-9]+).*/#\1 /"
    git log --merges Pacemaker-2.0.4..HEAD configure.ac

lists just three numbered PRs (the second also lists a merge without a PR#), but many more PRs affected that file.

Reply:
That is interesting. Were the other PRs merged without merge commit? 
Intention behind getting this list was the possibility to read through the PRs if in doubt what really happened.
So the merge is rather unwanted for that. But for a one-liner the grep needed would get rid of it as well and going for --merges definitely reduces the amount of stuff that has to be piped around and if reduces the risk that grepping for "Merge pull request" will give false positives.

Reply:
As an example, 997f1435 touched configure.ac and shows up in "git log Pacemaker-2.0.4..HEAD configure.ac" (i.e. without limiting to merges), but its merge commit 50e698c2 doesn't show up in our checks.